### PR TITLE
Fix force push to `python-eccodes`

### PR DESCRIPTION
**Pull request by jakirkham**
Was pointing to commit ( https://github.com/conda-forge/python-eccodes-feedstock/commit/18e70b118fd7e337c1b0de1d1ff0b5b58ffa2e71 ). However that commit appears to have been forcibly removed from history. So fix to commit ( https://github.com/conda-forge/python-eccodes-feedstock/commit/abb9f420481873fe4d1253436738fea16a7c3a0a ), which is current `master`.

### CODE REVIEW

**Comment by pelson** [comment]
@ocefpaf - any idea how there came to be a force push? In terms of transparency, I'm extremely keen not to lose commits if we can possibly help it (I think I'm preaching to the converted here 😄 🙏 ⛪️ )

**Comment by ocefpaf** [comment]
> In terms of transparency, I'm extremely keen not to lose commits if we can possibly help it (I think I'm preaching to the converted here 😄 🙏 ⛪️ )

Indeed.

> @ocefpaf - any idea how there came to be a force push? 

If I am reading the dates correctly this was around the time I was teaching a workshop for 20 MetOcean scientists and trying to convince them use conda-forge.

TL;DR I have no idea b/c I was running up and down students laptops fixing packages issue that day.

**Comment by jakirkham** [comment]
Thanks @pelson. FWIW this is a persistent problem. Have noted cases in issue ( https://github.com/conda-forge/conda-forge-webservices/issues/139 ). Some are merely mistaken pushes to `master` that people have quickly deleted afterwards. A small number are actual misuse (e.g. pushing to `master` until something passes). The latter is more concerning IMHO. We could block force pushes to branches via a GitHub setting if we want to be sticklers about it.
